src/Sarif.FunctionalTests/DirectProducerTestData/WebSkim/SimpleBundleDoubleNesting.sarif

```diff
@@ -320,61 +320,45 @@
           ]
         },
         {
-          "ruleId": "JS2027",
-          "kind": "warning",
-          "formattedMessage": {
-            "formatId": "Warning"
-          },
-          "locations": [
-            {
-              "analysisTarget": {
-                "uri": "file:///d:/src/SecDevTools/Main/src/ModernCop/Source/Mod50.Test.Unit/ModernCopTestData/Mod50/CompiledPackages/SimpleJSApp.appxbundle#/js/default.js",
-                "region": {
-                  "startLine": 1
-                }
-              }
-            }
-          ]
-        },
-        {
-          "ruleId": "JS2073",
+          "ruleId": "JS2074",
```

Comment:
yes, new issue: sarif log emits issues in a non-deterministic order due to lack of sorting